contents/docs/migrate/migrate-from-amplitude.mdx

```diff
@@ -4,6 +4,12 @@ sidebar: Docs
 showTitle: true
 ---
 
+> Prior to beginning a historical data import please ensure that you've done the following:
+> 1. Create a project on our [US](https://us.posthog.com/signup) or [EU](https://eu.posthog.com/signup) Cloud
+> 2. Sign up to a paid plan on the billing page in-app (historic imports are not billable but this will unlock the necessary features).
```

Comment:
```suggestion
> 2. Sign [up to a paid plan on the billing page](https://us.posthog.com/organization/billing) (historic imports are not billable but this will unlock the necessary features).
```